File: cmd_run.py
#!/usr/bin/env python

import subprocess

def cmd_run(cmd):
    proc = subprocess.Popen(
        cmd,
        stdout = subprocess.PIPE,
        stderr = subprocess.PIPE
    )
    stdout_value, stderr_value = proc.communicate()
    exit_value = proc.returncode
    retval = {
        'stdout': stdout_value,
        'stderr': stderr_value,
        'exit': exit_value
    }
    return retval

File: cmd_run.pl
#!/usr/bin/env perl

use strict;
use warnings;
use IPC::Run3;

sub run_cmd {
    my @cmd = @_;
    my $stderr;
    my $stdout;

    run3( \@cmd, undef, \$stdout, \$stderr );
    my $exit = $?;

    my %retval = (
        stdout => $stdout,
        stderr => $stderr,
        exit   => $exit,
    );

    return \%retval;
}
